README.md

```diff
@@ -287,7 +287,17 @@ Use to listen event when token is revoked.
 
 ```
 func onStatusChanged(error: Talkshoplive.APIClientError) {
```

Comment:
Can you change this to `onStatusChange` instead of `changed`? To keep it consistent with Android.